### ISSUE TITLE
WalletConnect issues after v0.25

### ISSUE THREAD

**Issue by ezxpro** [horizontalsystems/unstoppable-wallet-android#5029]
Not sure if this is a WalletConnect or Unstoppable issue. 
Tried connecting to some DeFi apps today, see.

ThorChain: 
![image](https://user-images.githubusercontent.com/15344931/173940980-73b43650-4c98-4d07-a6e0-26e5d07a3164.png)

Moonbeam: 
![image](https://user-images.githubusercontent.com/15344931/173941007-917868b0-a955-4de6-87de-430ad48a4cff.png)

OrionProtocol (on my end, this dApp is broken in every possible way, so idk):
![image](https://user-images.githubusercontent.com/15344931/173941244-0d06cce9-5d46-4cdf-a7d5-ec532aea4eee.png)

Others are working fine, like Biswap.

**Comment by omurovch** [comment]
@ezxpro, we have tested each dapp you mentioned. 
- ThorChain: WC worked normally, may be it had some issues on the dapp side then.
- Moonbeam: unstoppable doesn't support it yet, and the error message should be "Unsupported Blockchain". 
- OrionProtocol: WC worked for Ethereum, Binance Smart Chain and Polygon networks. Fantom is not supported. 

So correct error messages for each unsupported blockchain and invalid url were added.
